inductiva/stellarators/simulators/simsopt.py

```diff
@@ -0,0 +1,49 @@
+"""Simsopt module of the API."""
+from typing import Optional
+from uuid import UUID
+
+from inductiva import types, tasks
+from inductiva.simulation import Simulator
+
+
+class Simsopt(Simulator):
```

Comment:
I think we should adapt this name in the future to reflect the specific use-case we're using simsopt for but for now this should be fine.

Reply:
Noted!